ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -412,7 +412,7 @@ <h3 class="title">
           </ng-container>
 
           <ng-container matColumnDef="srcOrder">
-            <th class="table_header" mat-header-cell *matHeaderCellDef>Order</th>
+            <th class="table_header" mat-header-cell *matHeaderCellDef>S No.</th>
```

Comment:
Primary key tab shows `S. No` for source and `order` for spanner which is wrong. It should be order for both as column orders matter in PK.

Reply:
Fixed.